docs/en/credential-issuance-l2plus.rst

```diff
@@ -0,0 +1,671 @@
+
+Low-Level Flow
+--------------
+
+This section provides technical details on how to request PID Issuance through the combination of Level of Assurance Substantial electronic identification and remote identity proofing through verification of the electronic document. The MRTD Verification service handles secure document reading and validation as part of an eID Substantial Authentication with MRTD Verification process initiated by the PID Authorization Server.
+
+.. _fig_eID_MRTD_Detailed_Flow:
+.. plantuml:: plantuml/l2plus-detailed-flow.puml
+    :width: 99%
+    :alt: The figure illustrates the eID Substantial Authentication with MRTD Verification Detailed Flow.
+    :caption: `eID Substantial Authentication with MRTD Verification Detailed Flow. <https://www.plantuml.com/plantuml/svg/nLPhRzku4ltENp4tHNiJDqgQf1ykZBgfwZpYYpOHOkSMTpSBWwR6DhSKgIMfFFRNBuUIvKVNHO5TO073iAKpCsSUFCCFpdFhgqAnV__Bpx5046WhXLmdfl9AQcIyyaPNnGGjAmDM2bboxM7dpg7T0UxWRltv5wuKUXZeu5eBZ91q7T8VdCV7VJOnJm0xsU0CCciUP4ukNtoIixd_g2fSjo9KMlcviV9txgNHC4Jxq918X-lWpxUZuDDa4ICR6cVPJJPag7C8SR-X1DVHArTmYpFfl7sEHfy7r-lG1UnU3tBKNlhdbdmq69nbZB7mN25WuFp9f2UHE855pVmrM2oCHv0B5wKrPYhr3AI6mIXfgaWv3vohg99meSmZOtUGl8Uq2r-8Z3V0XPSFt2E40D2OtUlAaQ-YKbucpcC9V9r0WQ6GEMEzNW-oENS8Hrsu8PORDRx5tofq7dezbiR8G-X2bjwsBtO1X9B1zLZc-s3hfpUJNr7ulK04u57wUUqwwKTWadeVCBL_hlEjBnDgeDGniolaELg2ayzE9thbgrJ7ENek5VZd4c57-h4wVfqRKHN4BPPtfzDXB2IGK1vj6guqsY7iDjJ7bPNxW4-bjEZ6KkyjykvIDxPKXpXTde_WSC48JzkYz0uE3bwjEAV_XByopdUXb7bIDkDpW4-yA1KU63jRRSrn26rbmUtpvXmbSSBeA6w-eVhGwC6mcj19z9AhsCHXo8Pcj4kcwrPjFYoyXtTKfhjkxPrlpg-Wekc8hCkpnLzTtTLiJhhGdix_SINp26svqMnVZKRPuT714PoyEOOBKyMcVpArqoxClIzTz_0mYjV1OvpvrTeTYZazqpCyxNGAw_Ew_Rrtoz_Z_tuPlO_ZG26ZC7J3LBN4ca9qElSwknc6nc9kH19D3wMMlgKOY7s_aVVwxpm0Z9dICm28dAbU951T28IzbVY0eRO_1PEfiKLqGLY2GBBCSnE2EcUiyKSDMAr4sq8gHTkNuz2NL0WiFUPDWdLjJJdMHWlSXxO7CSDTKzQggPxtOlpGTSuOLnu6qvgG0sqyy0Sk5Pye16zYtXlx9EQJGXpDHPS8AWD5ww6VNaV4GYK7oyUe789iWvc9vr9Z3jp3a1JowAZrJPRdJzvouIDHc5fJa65TgfarLTdigV-Z2h8CrnVz4CiZcRJ7Z-Z7qnDA3MUD5456GmJXanddq3c2XsVU2AEYnXEB1PMVNymielxbFVIXRKjQbcgf05dQFwVct2BFuUpoQ9--mlVmvcnCyEYg3qdhAuhVdo61RGixI8UhKTjmruFXFgHqtfNHj76oQX90y1MVQxTak8N0Zzkk4-gk6SMYQwusbETUktWvM8xX5XQtmpGvVlFsF-3aJ7DVMQJM15TBIshwLg5OAjXtDIDGaTFdVqmr0DYwNX2H1M4QJ-f6qh-0UosCjQZgA6AEuckzSnyeJxgeK93ADKKA-CMAfGNwOCIAuHvZ6yApeXzluERhTj5PLlM7zSTYBYsAk6A9SYr8WVPIbAY9CLHizdS0iSKVhNbqQ9C9TvZ3XQGXwnizbRPe3q7O-UqssjRAGp4vZTvxtmOS4n7CNndQh-D63UeBmgwVMrFDvZ2fUUojxj2tsovhTQtMbccdG_UMBS1OkkYfR_9Oux-uUqy3MWHNTHBmYiQb_IETwqK7Bq_IbySNuRFz-cEryGY6pd2wdmw3t0ukhqQNj-VdrsmqHvYIKWgZlTGLEk1269kJ8dYpVcsEGEuWnsdK--PaB2_PhaGXfrAqop26PX_2hwfGVm00>`_
+
+Phase 1: OAuth Authorization Request
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Wallet Instance initiates the eID Substantial Authentication with MRTD Verification flow by submitting a Pushed Authorization Request (PAR) containing a JWT-secured Authorization Request (JAR) with authentication requirements specified via Rich Authorization Requests (RAR).
```

Comment:
```suggestion
The Wallet Instance initiates the eID Substantial Authentication with MRTD Verification flow by submitting a Pushed Authorization Request (PAR), using a JWT-secured Authorization Request (JAR) and Rich Authorization Requests (RAR) with the further authentication requirements defined in the section ...
```

please use an Href with the endpoint section definiting the flow and its requirements.